### DISCUSSION TITLE
Outputting control_unit_row_index control_unit_column_index as integers

### DISCUSSION THREAD

**Discussion by RibbleFripFnibbulator** [helgoboss/helgobox#1280] [Ideas]
Hello helgoboss, 

I've made use of the control_unit_row_index and control_unit_column_index for dynamic targeting of slots on a scrollable controller. Scrolling is working great, however I would like to be able to poll for these two variables and output the numbers via OSC, or ideally monitor them any time they change and output OSC feedback. Is there any way to do this?

Thanks

**Comment by helgoboss** [comment]
You could use the target "Playtime: Control unit scroll". This can expose the current scroll position using the usual ReaLearn feedback mechanism, that means as percentage (number between 0.0 and 1.0). If you need the absolute scroll position per axis (as an integer), I could add a target property to this target. In that case, please change the category of this discussion into "Idea".

**Reply by RibbleFripFnibbulator** [reply]
Thanks, yes, the integer value would be great. Changing to idea.
